### PR TITLE
sbt task to launch lldb

### PR DESCRIPTION
**Pull request by MasseGuillaume**

### CODE REVIEW

File: sbt-scala-native/src/main/scala/scala/scalanative/sbtplugin/ScalaNativePluginInternal.scala [hunk 1]
```diff
@@ -322,6 +322,16 @@ object ScalaNativePluginInternal {
...
+    nativeDebug := {
+      val binary = abs(nativeLink.value)
+      val lldb   = abs(nativeLLDB.value)
+      val logger = streams.value.log
+      Process(lldb +: Seq("-f", binary, "-o", "run"))
+        .run(logger, connectInput = true)
+        .exitValue
+
+      ()
+    },
```
Comment (Duhemm) [inline sbt-scala-native/src/main/scala/scala/scalanative/sbtplugin/ScalaNativePluginInternal.scala]: It would be cool to also recompile with `-g`.

Comment (densh) [inline sbt-scala-native/src/main/scala/scala/scalanative/sbtplugin/ScalaNativePluginInternal.scala]: It doesn't really matter as that flag only applies for the C/C++ code, not LLVM that we generate. We don't emit any debug symbols ourselves yet (see #29.)

File: sbt-scala-native/src/main/scala/scala/scalanative/sbtplugin/ScalaNativePluginInternal.scala [hunk 2]
```diff
@@ -322,6 +322,16 @@ object ScalaNativePluginInternal {
       IO.read(nativeNativelib.value / "target")
     },
     nativeMode := "debug",
+    nativeDebug := {
+      val binary = abs(nativeLink.value)
+      val lldb   = abs(nativeLLDB.value)
```
Comment (densh) [inline sbt-scala-native/src/main/scala/scala/scalanative/sbtplugin/ScalaNativePluginInternal.scala]: Code for discovery of `lldb` is missing.

File: sbt-scala-native/src/main/scala/scala/scalanative/sbtplugin/ScalaNativePluginInternal.scala (hunk 1 above)
Comment (Duhemm) [inline sbt-scala-native/src/main/scala/scala/scalanative/sbtplugin/ScalaNativePluginInternal.scala]: Wouldn't it still help for debugging the nativelib?

Comment (densh) [inline sbt-scala-native/src/main/scala/scala/scalanative/sbtplugin/ScalaNativePluginInternal.scala]: But you'll have to turn it off for release builds. I'm not sure if extra complexity is worth it here. Especially considering that C/C++ code in nativelib are mostly wrappers.

**Comment by densh** [comment]
Lets delay this one for now. Feel free to reopen once 0.2 is out. :)

**Pull request closed without merging**